packages/palette/package.json

```diff
@@ -22,8 +22,7 @@
     "release": "auto shipit && node scripts/clean-up-after-auto.js",
     "test": "yarn type-check && yarn jest --runInBand",
     "test:watch": "jest --watch --runInBand",
-    "type-check": "tsc --noEmit --pretty && yarn type-check-ios && yarn type-check-scripts",
-    "type-check-ios": "tsc -p tsconfig.ios.json --noEmit --pretty",
+    "type-check": "tsc --noEmit --pretty && yarn type-check-scripts",
     "type-check-scripts": "tsc -p tsconfig.scripts.json --noEmit --pretty",
     "type-declarations": "tsc --emitDeclarationOnly",
     "watch": "concurrently --raw --kill-others 'yarn compile -w' 'yarn type-declarations -w'",
```

Comment:
you should also delete the line below this (github won't let me add a comment there 🙈 )

Reply:
there's also the `"clean:eigen"` and `"compile:eigen"` commands in this file

Reply:
sure 🎉